kpops/cli/utils.py

```diff
@@ -25,7 +25,10 @@ def collect_pipeline_paths(pipeline_paths: Iterable[Path]) -> Iterator[Path]:
             # docs.python.org/3.13#pathlib.Path.glob, probably it make sense to use it after ugprading,
             # likely the code will look like:
             # yield from sorted(pipeline_path.glob(f"**/{PIPELINE_YAML}", recurse_symlinks=True))
-            yield from sorted(Path(p).resolve() for p in glob(f"{pipeline_path}/**/{PIPELINE_YAML}", recursive=True))
+            yield from sorted(
+                Path(p).resolve()
+                for p in glob(f"{pipeline_path}/**/{PIPELINE_YAML}", recursive=True)  # noqa: PTH207
+            )
```

Comment:
Perhaps actually extract a method here, e.g. for the `glob()` call

Reply:
good point, but I am rather hoping to resolve this with the todo comment above